Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE=hexpm/elixir:1.14.2-erlang-24.3.4.2-debian-bullseye-20210902-slim
+ARG BASE_IMAGE=hexpm/elixir:1.15.0-rc.1-erlang-25.3.2-debian-bullseye-20230522-slim
```

Comment:
Still waiting for the amd variants of this [image](https://hub.docker.com/r/hexpm/elixir/tags?page=1&name=1.15.0-rc.1-erlang-25.3.2-debian-bullseye).